packages/replay/src/replay.ts

```diff
@@ -450,6 +539,30 @@ export class ReplayContainer implements ReplayContainerInterface {
     this._context.urls.push(url);
   }
 
+  /**
+   * Initialize and start all listeners to varying events (DOM,
+   * Performance Observer, Recording, Sentry SDK, etc)
+   */
+  private _initializeRecording(): void {
+    this.setInitialState();
+
+    // this method is generally called on page load or manually - in both cases
+    // we should treat it as an activity
+    this._updateSessionActivity();
+
+    this.eventBuffer = createEventBuffer({
+      useCompression: this._options.useCompression,
+    });
+
+    // TODO: we should probably remove listeners before adding listeners
+    this._addListeners();
```

Comment:
I guess we can just always call `this._removeListeners()` here, I think that should be idempotent and work fine if running "empty"?